.github/workflows/build-and-publish.yml

```diff
@@ -23,22 +23,15 @@ jobs:
           - registry: dockerhub
             platform: linux/riscv64
             alpine_version: edge
-            platform_dir: linux-riscv64
           - registry: ghcr
             platform: linux/riscv64
             alpine_version: edge
-            platform_dir: linux-riscv64
-          - platform: linux/amd64
-            platform_dir: linux-amd64
-          - platform: linux/386
-            platform_dir: linux-386
-          - platform: linux/arm/v6
-            platform_dir: linux-arm-v6
-          - platform: linux/arm/v7
-            platform_dir: linux-arm-v7
-          - platform: linux/arm64
-            platform_dir: linux-arm64
     steps:
+      - name: Prepare name for digest up/download
+        run: |
+          platform=${{ matrix.platform }}
+          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
```

Comment:
does `PLATFORM_PAIR` get used anywhere?

Reply:
Good catch. I forgot to add it.

Reply:
It should now follow https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners